config/samples/deployment.yaml

```diff
@@ -122,6 +122,9 @@ spec:
       containers:
       - command:
         - /manager
+        args:
+        - "-watch-namespace"
+        - "$(WATCH_NAMESPACE)"
```

Comment:
I didn't know Kubernetes had first-class support for variable substitution in the args. That's neat!